### DISCUSSION TITLE
How to transform an AST node to a node with different type?

### DISCUSSION THREAD

**Discussion by Utakata-t** [swc-project/swc#5577] [Q&A] [answered]
I am trying writing a plugin to replace an AST node with another node of a different type

for example:

`//input: 
 import "./color.less";
//output:
require("./color.less");
`

the type of the input node is 'ImportDeclaration',  but the output is a 'ExpressionStatement'.

How can I do this transforming?

**Comment by daflyinbed** [comment, marked as answer]
visit Module and change its body

**Reply by Utakata-t** [reply]
thanks!！Module is  the type I am looking for.
